src/collection_test.go

```diff
@@ -180,8 +190,19 @@ func Test_collectMetrics(t *testing.T) {
 	}
 
 	assert.Equal(t, float64(1.0), frontendEntity.Metrics[0].Metrics["frontend.currentSessions"])
+
 	assert.Equal(t, float64(1.0), backendEntity.Metrics[0].Metrics["backend.currentSessions"])
+	assert.Equal(t, float64(0.01), backendEntity.Metrics[0].Metrics["backend.averageQueueTimeInSeconds"])
+	assert.Equal(t, float64(0.011), backendEntity.Metrics[0].Metrics["backend.averageConnectTimeInSeconds"])
+	assert.Equal(t, float64(0.012), backendEntity.Metrics[0].Metrics["backend.averageResponseTimeInSeconds"])
+	assert.Equal(t, float64(0.013), backendEntity.Metrics[0].Metrics["backend.averageTotalSessionTimeInSeconds"])
+
 	assert.Equal(t, float64(1.0), serverEntity.Metrics[0].Metrics["server.currentSessions"])
+	assert.Equal(t, float64(0.1), serverEntity.Metrics[0].Metrics["server.averageQueueTimeInSeconds"])
+	assert.Equal(t, float64(0.101), serverEntity.Metrics[0].Metrics["server.averageConnectTimeInSeconds"])
+	assert.Equal(t, float64(0.102), serverEntity.Metrics[0].Metrics["server.averageResponseTimeInSeconds"])
+	assert.Equal(t, float64(0.103), serverEntity.Metrics[0].Metrics["server.averageTotalSessionTimeInSeconds"])
+	assert.Equal(t, float64(0.104), serverEntity.Metrics[0].Metrics["server.agentDurationSeconds"])
```

Comment:
Is it `agentDurationSeconds` or `agentDurationInSeconds`?

Reply:
It is `agentDurationSecons` (names are not changed, we decided to fix the values but keep the names as they were). No idea why this the only one without `In`.